libparsec/crates/platform_device_loader/src/lib.rs

```diff
@@ -264,3 +264,41 @@ pub enum RemoveDeviceError {
 }
 
 pub use platform::remove_device;
+
+#[derive(Debug, thiserror::Error)]
+pub enum PlatformImportRecoveryDeviceError {
+    #[error(transparent)]
+    InvalidPath(anyhow::Error),
+    #[error("Cannot deserialize file content")]
+    InvalidData,
+    #[error("Passphrase format is invalid")]
+    InvalidPassphrase,
+    #[error("Failed to decrypt file content")]
+    DecryptionFailed,
+    #[error(transparent)]
+    SaveDeviceError(#[from] SaveDeviceError),
+}
+
+#[derive(Debug, thiserror::Error)]
+pub enum PlatformExportRecoveryDeviceError {
+    #[error(transparent)]
+    Internal(#[from] anyhow::Error),
+    #[error(transparent)]
+    LoadDeviceError(#[from] LoadDeviceError),
+}
+
+pub async fn inner_import_recovery_device(
+    recovery_device: Vec<u8>,
+    passphrase: SecretKeyPassphrase,
+    device_label: DeviceLabel,
+) -> Result<(LocalDevice, LocalDevice), PlatformImportRecoveryDeviceError> {
+    platform::import_recovery_device(recovery_device, passphrase, device_label).await
+}
+
+/// returns (secret key passphrase, recovery device encrypted data, recovery device)
+pub async fn inner_export_recovery_device(
+    device: &LocalDevice,
+    device_label: DeviceLabel,
+) -> Result<(SecretKeyPassphrase, Vec<u8>, LocalDevice), PlatformExportRecoveryDeviceError> {
+    platform::export_recovery_device(device, device_label).await
+}
```

Comment:
They don't need specific platform impl (or don't seems to), move them directly here